test/Serilog.Settings.Configuration.Tests/ConfigurationSettingsTests.cs

```diff
@@ -12,9 +12,12 @@ namespace Serilog.Settings.Configuration.Tests
 {
     public class ConfigurationSettingsTests
     {
-        static LoggerConfiguration ConfigFromJson(string jsonString)
+        static LoggerConfiguration ConfigFromJson(string jsonString, string secondJsonSource = null)
```

Comment:
could possibly make this `params` array, but rule of threee

Reply:
And YAGNI... I can't think of a test scenario where 'n' config sources would do anything two sources can't represent.